### PR TITLE
Fix missing semicolon in Windows-specific code in aigerparse.cc.

### PR DESCRIPTION
**Pull request by cr1901**
Fixes a compile error on Windows, tested using `msys2`. No other platforms affected.

### CODE REVIEW

File: frontends/aiger/aigerparse.cc [hunk 1]
```diff
@@ -1062,7 +1062,7 @@ struct AigerFrontend : public Frontend {
 		if (module_name.empty()) {
 #ifdef _WIN32
 			char fname[_MAX_FNAME];
-			_splitpath(filename.c_str(), NULL /* drive */, NULL /* dir */, fname, NULL /* ext */)
+			_splitpath(filename.c_str(), NULL /* drive */, NULL /* dir */, fname, NULL /* ext */);
 				module_name = fname;
```
Comment (eddiehung) [inline frontends/aiger/aigerparse.cc]: Oops. Looks like I did an extra tab here. Would you be able fix that too whilst you're in the region? Thanks!

**Comment by eddiehung** [comment]
Thanks. Will merge when CI goes green, not that CI even builds on Windows anymore (I think it was having issues in the past; we should really fix the underlying cause not by disabling stuff!)

**Comment by cr1901** [comment]
>(I think it was having issues in the past; we should really fix the underlying cause not by disabling stuff!)

I would fix it myself, but Travis currently [doesn't play nice](https://travis-ci.community/t/how-do-i-use-mingw-and-msys2-shells/1787) w/ `msys2`, and IME that is the only way to really get a `yosys` on Windows that supports everything (as opposed to mxe).

Maybe PR #747 would give me an opportunity to test this? On my radar is _also_ to make official packages for `msys2` for `yosys-git`, `nextpnr-git`, and `yices2` (`z3` is already there!).

**Comment by eddiehung** [comment]
> Maybe PR #747 would give me an opportunity to test this?

My impression is that PR#747 doesn't has fallen off the radar and no-one is driving it forward anymore. Would you like to adopt it this poor orphan? We use Cirrus in [nextpnr](http://github.com/YosysHQ/nextpnr)...

**Pull request merged**